astroid/rebuilder.py

```diff
@@ -135,7 +135,7 @@ def _get_context(
             "ast.Tuple",
         ],
     ):  # TODO return type needs change to _Context enum
-        return self._parser_module.context_classes.get(type(node.ctx), astroid.Load)
```

Comment:
Would you mind fixing the `TODO` here as well? I put it there for exactly the time we refactor `Context` 😅 It should probably be: `-> Context`

Reply:
Nice catch. Yeah I thought that deleting the temporary enum would prevent from using it as Typing; Very strange design decision (probably taken in python 2's days).